### PR TITLE
Add support for tbd libraries

### PR DESCRIPTION
**Pull request by raptorxcz**
issue: #379.

### CODE REVIEW

**Comment by raptorxcz** [comment]
More at http://www.vadimbulavin.com/static-dynamic-frameworks-and-libraries/.

**Comment by CocoaPodsBarista** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">Please include a CHANGELOG entry to credit yourself! <br />
You can find it at <a href="https://github.com/CocoaPods/CocoaPods/blob/master/CHANGELOG.md">CHANGELOG.md</a>.</td>
    </tr>
  </tbody>
</table>

Here's an example of your CHANGELOG entry:

```markdown
* Add support for tbd libraries  
  [raptorxcz](https://github.com/raptorxcz)
  [#issue_number](https://github.com/CocoaPods/CocoaPods/issues/issue_number)
```

*note*: There are two invisible spaces after the entry's text.


<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">Danger</a>
</p>

**Comment by dnkoutso** [comment]
LGTM can you rebase squash?

File: lib/xcodeproj/project/object/native_target.rb [hunk 1]
```diff
@@ -377,6 +377,27 @@ def add_system_library(names)
         end
         alias_method :add_system_libraries, :add_system_library
 
+        # Adds a file reference for one or more system libraries to the project
```
Comment (dnkoutso) [inline lib/xcodeproj/project/object/native_target.rb]: Does this doc need to be updated? 

Also can we extract common logic between this method and `add_system_library` method?

**Comment by dnkoutso** [comment]
Also needs a `CHANGELOG` entry please.

**Comment by raptorxcz** [comment]
Fixed, thanks a lot.

**Comment by dnkoutso** [comment]
We do not do releases on demand, you can use https://bundler.io/v1.12/git.html to point to the master or a SHA which includes your fix. I believe a release would occur within the next 1-4 weeks.

**Pull request merged**